packages/fuels/src/run.ts

```diff
@@ -1,6 +1,8 @@
 import { configureCli } from './cli';
+import { checkForAndDisplayUpdates } from './cli/utils/checkForAndDisplayUpdates';
+import { error } from './cli/utils/logger';
 
 export const run = async (argv: string[]) => {
   const program = configureCli();
-  return program.parseAsync(argv);
+  return Promise.all([await checkForAndDisplayUpdates().catch(error), program.parseAsync(argv)]);
```

Comment:
@Dhaiwat10 This could potentially be simplified to:

```ts
  await checkForAndDisplayUpdates().catch(error)
  const program = configureCli();
  return program.parseAsync(argv);
```